cmd/kube-apiserver/app/options/options.go

```diff
@@ -180,6 +182,7 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 	s.Admission.AddFlags(fss.FlagSet("admission"))
 	s.Metrics.AddFlags(fss.FlagSet("metrics"))
 	s.Logs.AddFlags(fss.FlagSet("logs"))
+	s.OpenTelemetry.AddFlags(fss.FlagSet("opentelemetry"))
```

Comment:
> we should probably eventually refactor this into flags defined from component-base/traces (but not necessarily right now)

As per @logicalhan  comment above, should we not call this `traces` instead of `opentelemetry`. We can always wrap later but this is user facing no?

Reply:
Agreed. We should be consistent here and call this `traces` as opposed to `opentelemetry`.

Reply:
Changed all references in the PR to use `tracing`/`traces` instead of `opentelemetry`.  I also updated to KEP to clarify this in https://github.com/kubernetes/enhancements/pull/2526.

Reply:
Thank you!